### DISCUSSION TITLE
I just forked Fresh and added stuff for my needs

### DISCUSSION THREAD

**Discussion by juanvillacortac** [freshframework/fresh#874] [Ideas]
Hi to everyone. I want to share my last tiny side project, a little Fresh fork where I implement an alternative way to use it, with things like:

* Islands on any directory, only suffix the component with .island.{tsx|ts|jsx|js}.
* File-system routing à la SvelteKit, including nested layouts on N levels (you can access handlers global state here!)

![image](https://user-images.githubusercontent.com/37244056/199349969-b5c48cf2-416f-4a88-aa97-1aefcd71058b.png)

I share this because I think that my Fresh implementation can be useful for the main Fresh development.

Check the [main repo](https://github.com/juanvillacortac/citric) or start a new Citric project running `deno run -A -r https://citric.deno.dev deno-citric-demo`

**Comment by reggi** [comment]
islands files ++ can we get this in fresh? 🙏
